core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.spec.ts

```diff
@@ -1220,7 +1220,8 @@ describe('Contributions and review component', () => {
             toHaveBeenCalledWith(
               suggestion1,
               'suggestion_1',
-              true
+              true,
+              []
```

Comment:
Can we have a test where `skippedContributionIds` is not empty?

Reply:
Done